### PR TITLE
Added html for section 4

### LINKED ISSUE #7: html section 4
We need to create the layout of the section 4

### PR DESCRIPTION
**Pull request by AlekseyBelousov**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -57,7 +57,17 @@ <h2 class="care__title">ZEN Care</h2>
       </div>
     </section>
     <!-- Секция 4 -->
-    <section>Секция 4</section>
+    <section>
```
Comment (AlekseyBelousov) [inline index.html]: Need add class name "cashback" and id name "cashback"

File: index.html [hunk 2]
```diff
@@ -57,7 +57,17 @@ <h2 class="care__title">ZEN Care</h2>
       </div>
     </section>
     <!-- Секция 4 -->
-    <section>Секция 4</section>
+    <section class="cashback" id="cashback">
+      <div class="cashback">
```
Comment (nagorapro) [inline index.html]:
```suggestion
      <div class="cashback__wrapper">
```

**Review by nagorapro — COMMENTED** [review]
Need fix

**Review by nagorapro — APPROVED** [review]
Good

**Pull request merged**